public_suffix_list.dat

```diff
@@ -472,26 +479,36 @@ jor.br
 jus.br
 leg.br
 lel.br
+londrina.br
+macapa.br
+maceio.br
 mat.br
 med.br
 mil.br
 mp.br
 mus.br
+natal.br
 net.br
+niteroi.br
 *.nom.br
 not.br
 ntr.br
 odo.br
 org.br
+osasco.br
+palmas.br
 poa.br
 ppg.br
 pro.br
 psc.br
 psi.br
+pvh.br
```

Comment:
I cannot find supporting documentation for this on the https://registro.br/dominio/categoria.html page